modules/backend/behaviors/RelationController.php

```diff
@@ -1157,10 +1157,8 @@ public function onRelationManageUpdate()
             }
         }
         elseif ($this->viewMode == 'single') {
-            $this->viewModel = $this->manageWidget->model;
```

Comment:
Why would you just change this line to `$this->viewWidget->model`?

Reply:
You right, I made many tests and this was the latest working code. Let me change it.